### PR TITLE
feat(embodiment): add LingBot-VA SFT + eval on Libero-Object

### PR DESCRIPTION
**Pull request by ZiruiZhou**
### Description

End-to-end integration of the LingBot-VA video-diffusion VLA into RLinf for supervised fine-tuning and evaluation on Libero-Object.

**SFT recipe (8 × A100-80GB, single node):**
- `examples/sft/config/libero_sft_lingbotva.yaml`: bf16 + FSDP2 full-shard + plain torch AdamW + flex_attention. micro=1, global_batch_size=80 (grad_accum=10 on ws=8), matching the lingbot-va reference `va_libero_train_cfg` effective batch. Steady-state ~12.5 s/step at ~32 GB/GPU.
- Single-GPU launch (L40-class, ≥45 GB) is reached via CLI overrides (FSDP1 + NO_SHARD + bnb AdamW8bit + global_batch_size=10) and is documented inline in the yaml header.

**Evaluation:**
- `examples/embodiment/eval_lingbotva.py`: dedicated Hydra-driven single-process driver. Walks the chunk-step loop manually so it can feed `env.current_raw_obs` per step back through the model's KV-cache replay hooks. Going through the generic `MultiStepRolloutWorker` drops per-step obs and silently collapses SR to ~0 %.
- `examples/embodiment/eval_embodiment.sh` now dispatches the eval driver by `model_type`; other policies keep using `eval_embodied_agent.py`.
- Multi-GPU evaluation is one-task-per-GPU via `CUDA_VISIBLE_DEVICES` + `MUJOCO_EGL_DEVICE_ID` (~39 GB peak per process — two tasks do not fit on one 80 GB card).

**Data preparation:** Three conversion scripts vendored under `toolkits/data_scripts_lingbotva/`: raw HDF5 → LeRobot v2.1, deterministic per-task subset, and Wan VAE + UMT5 latent extraction. The lingbot-va peer repo is still needed at runtime for the `wan_va` Python package.

**Documentation:** `docs/source-en/rst_source/examples/embodied/sft_lingbotva.rst` covers setup, data prep, key config knobs, expected loss curve, the 8-GPU + single-GPU launch paths, parallel evaluation across 8 ranks, per-task verified SR, and common failure modes.

### Motivation and Context

Adds LingBot-VA as a supported embodied SFT model in RLinf, with a verified recipe on Libero-Object. This is the first video-diffusion VLA integration in RLinf — chunk-based action generation and KV-cache replay require a dedicated eval driver, hence the new `eval_lingbotva.py`.

### How has this been tested?

**SFT training (8 × A100-80GB, this branch):** 2000 steps in ~7 h. Loss trajectory matches the lingbot-va single-A100 reference at every step where the reference reports a number:

| Step | latent_loss | action_loss | grad_norm |
|---|---|---|---|
| 500 | ~0.120 | ~0.097 | ~0.077 |
| 1000 | ~0.113 | ~0.087 | ~0.091 |
| 1500 | ~0.103 | ~0.082 | ~0.111 |
| 2000 | ~0.103 | ~0.082 | ~0.121 |

**Evaluation (ckpt_2000, 10 Libero-Object tasks × 5 episodes = 50 episodes):**

| Model | Mean SR |
|---|---|
| `lingbot-va-base` (no SFT) | **0 % (0/50)** |
| SFT ckpt_2000 (this branch) | **70 % (35/50)** |

Per-task SR vector for SFT: `[3, 4, 5, 1, 4, 3, 5, 3, 4, 3] / 5`.

**E2E config:** `tests/e2e_tests/sft/libero_sft_lingbotva.yaml` added.

### Additional information (optional, e.g., figures and logs):

Full reproduction walkthrough (env, data prep, training, evaluation, common failure modes) in the new doc page at `docs/source-en/rst_source/examples/embodied/sft_lingbotva.rst`.

### Types of changes
- [x] New feature (non-breaking change which adds functionality)
- [x] Documentation update (Document-only update)

### Checklist:
- [x] My code follows the code style of this project.
- [x] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [x] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: rlinf/models/embodiment/lingbotva/eval_adapter/native_backend.py [hunk 1]
```diff
@@ -0,0 +1,726 @@
+# Copyright 2026 The RLinf Authors.
```
Comment (F9rozen) [inline rlinf/models/embodiment/lingbotva/eval_adapter/native_backend.py:1]: This file (~726 LOC) largely re-implements `wan_va/wan_va_server.py`
Notably, `_build_server()` already constructs `VA_Server`, but the public API then bypasses `server.infer()` and talks to `server.transformer` directly. That means RLinf now owns two copies of the diffusion loop and must manually stay in sync with upstream bugfixes (guidance, scheduler padding, cache semantics, etc.).
maybe：
```python
# Pseudocode — thin eval wrapper
for env_idx, (obs, prompt) in enumerate(zip(obs_batch, prompts)):
    server.infer({"reset": True, "prompt": prompt})
    for key_frames, prev_action in kv_histories[env_idx]:
        server.infer({"compute_kv_cache": True, "obs": key_frames, "state": prev_action})
    out = server.infer({"obs": current_obs})
